src/routes/root/root.controller.spec.ts

```diff
@@ -15,6 +17,8 @@ describe('Root Controller tests', () => {
     })
       .overrideModule(CacheModule)
       .useModule(TestCacheModule)
+      .overrideModule(configurationModule)
+      .useModule(ConfigurationModule.register(configuration))
```

Comment:
The Tenderly API is required and if we don't mock the configuration here, this test does not pass.